examples/kotlin-matter-controller/java/src/com/matter/controller/commands/pairing/PairingCommand.kt

```diff
@@ -162,6 +162,10 @@ abstract class PairingCommand(
     logger.log(Level.INFO, "onCommissioningStatusUpdate")
   }
 
+  override fun onCommissioningStageStart(nodeId: Long, stage: String?) {
+    logger.log(Level.INFO, "onCommissioningStageStart")
```

Comment:
should we log the stage as well? That seems useful info and maybe we should add the stage to the update as well.